harness/tests/addons/gut/plugin_control.gd

```diff
@@ -125,26 +125,26 @@ var _cancel_import = false
 var _placeholder = null
 
 func _init():
-    # This min size has to be what the min size of the GutScene's min size is
-    # but it has to be set here and not inferred i think.
-    rect_min_size = Vector2(740, 250)
+	# This min size has to be what the min size of the GutScene's min size is
```

Comment:
I think we should define what indentation we use for `gd` files. If i saw that correctly @raniejade uses spaces and @piiertho  and I use tabs.
What should we use?

Reply:
I don't have strong opinion about this.

Reply:
Me neither. I guess let's stick to the default of Godot which is tabs IIRC. WDYT @raniejade ?